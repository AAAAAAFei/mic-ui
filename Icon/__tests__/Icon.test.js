import React from 'react';
import renderer from 'react-test-renderer';

import Icon from '../Icon';

describe('Radio.List 快照测试', () => {
    it('renders correctly basic', () => {
        const tree = renderer.create(
            <Icon type={Icon.heart} />
        ).toJSON();
        expect(tree).toMatchSnapshot();
    });

    it('renders correctly with size="xl"', () => {
        const tree = renderer.create(
            <Icon type={Icon.heart} size="xl" />
        ).toJSON();
        expect(tree).toMatchSnapshot();
    });

    it('renders correctly with color="#ff571a"', () => {
        const tree = renderer.create(
            <Icon type={Icon.heart} color="#0000ff" />
        ).toJSON();
        expect(tree).toMatchSnapshot();
    });
});