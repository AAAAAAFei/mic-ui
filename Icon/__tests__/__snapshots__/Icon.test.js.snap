// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Radio.List 快照测试 renders correctly basic 1`] = `
<Image
  source={
    Object {
      "uri": "https://assets.souche.com/assets/sccimg/common/icon/toastcollection.png",
    }
  }
  style={
    Array [
      Object {},
      Object {
        "height": 16,
        "width": 16,
      },
      undefined,
    ]
  }
/>
`;

exports[`Radio.List 快照测试 renders correctly with color="#ff571a" 1`] = `
<Image
  source={
    Object {
      "uri": "https://assets.souche.com/assets/sccimg/common/icon/toastcollection.png",
    }
  }
  style={
    Array [
      Object {
        "tintColor": "#0000ff",
      },
      Object {
        "height": 16,
        "width": 16,
      },
      undefined,
    ]
  }
/>
`;

exports[`Radio.List 快照测试 renders correctly with size="xl" 1`] = `
<Image
  source={
    Object {
      "uri": "https://assets.souche.com/assets/sccimg/common/icon/toastcollection.png",
    }
  }
  style={
    Array [
      Object {},
      Object {
        "height": 24,
        "width": 24,
      },
      undefined,
    ]
  }
/>
`;
